packages/client/src/EditOnGitHubButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { Button } from '@chakra-ui/react';
+
+const GitHubIcon: FC<{ className?: string; size?: number }> = ({ className, size = 24 }) => {
```

Comment:
https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/

Reply:
```suggestion
const GitHubIcon = ({ className, size = 24 }: { className?: string; size?: number }) => {
```

Reply:
but now this fn don't return ReactElement :)

Reply:
Its inferred, but you can also add it explicitly.

```suggestion
const GitHubIcon = ({ className, size = 24 }: { className?: string; size?: number }): React.ReactElement => {
```